tests/smoke/smoke_test.go

```diff
@@ -0,0 +1,138 @@
+	"github.com/rs/zerolog/log"
+)
+
+func getMatchDir(policy types.KubeArmorPolicy, sys string, str string) string {
+	if sys == "file" {
+		for i := range policy.Spec.File.MatchDirectories {
+			if policy.Spec.File.MatchDirectories[i].Dir == str {
+				value := str
+				return value
+			}
+		}
+	} else {
+		for i := range policy.Spec.Process.MatchDirectories {
+			if policy.Spec.Process.MatchDirectories[i].Dir == str {
+				value := str
+				return value
+			}
+		}
+	}
+	return ""
+}
+
+func getMatchPath(policy types.KubeArmorPolicy, sys string, str string) string {
+	if sys == "file" {
+		for i := range policy.Spec.File.MatchPaths {
+			if policy.Spec.File.MatchPaths[i].Path == str {
+				value := str
+				return value
+			}
+		}
+	} else {
+		for i := range policy.Spec.Process.MatchPaths {
+			if policy.Spec.Process.MatchPaths[i].Path == str {
+				value := str
+				return value
+			}
+		}
+	}
+	return ""
+}
+
+func getWpsqlPod(name string, ant string, ns string) {
+	pods, err := util.K8sGetPods(name, ns, []string{ant}, 60)
+	Expect(err).To(BeNil())
+	Expect(len(pods)).To(Equal(1))
+}
+
+var _ = BeforeSuite(func() {
+	// install discovery-engine
+	_, err := util.Kubectl(fmt.Sprintf("apply -f https://raw.githubusercontent.com/kubearmor/discovery-engine/dev/deployments/k8s/deployment.yaml"))
+	Expect(err).To(BeNil())
+	time.Sleep(5 * time.Second)
+
+	//install wordpress-mysql app
+	err = util.K8sApply([]string{"res/wordpress-mysql-deployment.yaml"})
+	Expect(err).To(BeNil())
+	time.Sleep(5 * time.Second)
+
+	// delete all KSPs
+	err = util.DeleteAllKsp()
+	Expect(err).To(BeNil())
+
+	// enable kubearmor port forwarding
+	err = util.KubearmorPortForward()
+	Expect(err).To(BeNil())
+})
+
+var _ = AfterSuite(func() {
+	util.KubearmorPortForwardStop()
+})
+
+func discover(ns string, l string) (types.KubeArmorPolicy, error) {
+	var policy types.KubeArmorPolicy
+	cmd, err := exec.Command("karmor", "discover", "-n", ns, "-l", l, "-f", "json").Output()
+	if err != nil {
+		log.Error().Msgf("Failed to apply the `karmor discover` command : %v", err)
+	}
+	err = json.Unmarshal(cmd, &policy)
+	if err != nil {
+		log.Error().Msgf("Failed to unmarshal the policy : %v", err)
+	}
+	return policy, err
+}
+
+var _ = Describe("Smoke", func() {
+
+	BeforeEach(func() {
+		getWpsqlPod("wordpress-",
+			"container.apparmor.security.beta.kubernetes.io/wordpress: localhost/kubearmor-wordpress-mysql-wordpress-wordpress", "wordpress-mysql")
+		getWpsqlPod("mysql-",
+			"container.apparmor.security.beta.kubernetes.io/mysql: localhost/kubearmor-wordpress-mysql-mysql-mysql", "wordpress-mysql")
+		getWpsqlPod("discovery-engine-",
+			"container.apparmor.security.beta.kubernetes.io/discovery-engine: localhost/kubearmor-accuknox-agents-discovery-engine-discovery-engine", "accuknox-agents")
+	})
+
+	AfterEach(func() {
+		util.KarmorLogStop()
+		err := util.DeleteAllKsp()
+		Expect(err).To(BeNil())
+		// wait for policy deletion
+		time.Sleep(5 * time.Second)
```

Comment:
it's not needed here, we are not applying any security policy, 
is there any other purpose for the sleep here?

Reply:
I thought if someone had a kubearmor policy already in the cluster then it'll be removed. Should I remove this?